src/plugin-slots/CourseAuthoringUnitSidebarSlot/index.tsx

```diff
@@ -10,11 +10,14 @@ export const CourseAuthoringUnitSidebarSlot = (
     blockId,
     courseId,
     unitTitle,
+    verticalBlocks,
```

Comment:
I know the term `vertical` has been used internally for a long time, but I'm not sure if it's the public-facing name for this. By making it part of `pluginProps` we're "committing" to that as an "official" name by making it part of the explicit API.

I don't know if there's a name we'd prefer here. @arbrandes thoughts?

Reply:
@brian-smith-tcril @arbrandes we need these fixes for incontext-metric so it will be great if we can wind this up before teak cut.

Reply:
There's always been a dichotomy with the naming of the "unit" concept.  In the OLX itself it's "vertical", so we'll likely never be rid of the name.  However, since the prop immediately above it is `unitTitle`, it would make sense to align both to use the same terminology.

Reply:
@farhaanbukhsh @arbrandes I just realized something. These are actually verticalBlock's children. So these would be just `xblocks`. Correcting it now.

Reply:
I think it means "vertical blocks" as in "the blocks in the vertical".

> In the OLX itself it's "vertical", so we'll likely never be rid of the name.

In learning core and libraries, we are moving to use the term "unit" everywhere in the code and UI, except for OLX.